Formula/ponysay.rb

```diff
@@ -26,6 +30,6 @@ def install
   end
 
   test do
-    system "#{bin}/ponysay", "-A"
+    system "#{bin}/ponysay", "test"
```

Comment:
Can we check this for some output to make sure it's working?

Reply:
that's weird, it's happen with previous `system "#{bin}/ponysay", "-A"` too.
Do I write test incorrectly?

Here is the comparison when I run `brew test` and directly call it with full path same as the output from `brew test`
![diff](https://i.imgur.com/kCKwcMp.png)

Reply:
It's not incorrect, I'm just thinking of ways we can further improve the test. This could work
```suggestion
    assert_match "test", shell_output("#{bin}/ponysay test")
```

Reply:
oh you mean it like that, yeah, actually I already did that in my local 😅 I thought it would show some output like I did it directly.
Thank you.

Reply:
It only outputs if it fails